test/compilable/extra-files/test19266.d

```diff
@@ -0,0 +1,3 @@
+module test19266;
+
+pragma(msg, __FILE__);
```

Comment:
Is this doing anything for the test?

Reply:
No, but you can look it up in the test_result folder. If `\\?\` is given on the command line it is also found in `__FILE__`.